sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -816,6 +851,45 @@ public partial class JobResponse
         public static bool operator !=(Azure.Iot.Hub.Service.Models.JobResponseType left, Azure.Iot.Hub.Service.Models.JobResponseType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class MessageFeedbackBatch
+    {
+        internal MessageFeedbackBatch() { }
+        public System.DateTimeOffset? EnqueuedTime { get { throw null; } }
+        public string LockToken { get { throw null; } }
```

Comment:
Can we use BatchID as the name as oppose to locktoken? from our discussion yesterday it makes more sense.

Reply:
sure, will do

Reply:
in case this ID has an expiration time, then batch Id might not be the best name for it though

Reply:
I'll hold off until we get a confirmation on that from the service team, and update it then

Reply:
If we change the name, would the MSDN docs not be inconsistent? I believe is is already part of their GA api right?

Reply:
Surprisingly, this locktoken is not a part of any msdn docs: https://github.com/Azure/azure-sdk-for-net/pull/13489/files#r455909115